libs/product/driver/magento/src/queries/fragments/product-preview.ts

```diff
@@ -0,0 +1,30 @@
+import { gql } from 'apollo-angular';
+
+
+export const magentoProductPreviewFragment = gql`
+  fragment magentoProductPreview on ProductInterface {
```

Comment:
Does this type need the composite and configurable fragments?

Reply:
yes, added

Reply:
* added to the main product fragment, not to the preview fragment but the resulting query is equivalent

Reply:
wouldn't we want these on the preview fragment? That way, the configurable and bundle fields will be on both previews and products. And we'll want those fields for upsell and related products.